### PR TITLE
Implementation of multiple labels support

### PR DESCRIPTION
**Pull request by antonovchtd**
Before: label was a string in proto,
Now: repeated string.
Model API: updated TaskManager::AddLabel, created new TaskManager::ClearLabel, TaskManager::ClearLabels.

Thee new UI commands: "unlabel", "UNLABEL" (to remove all labels), and "labels" to show labels of a task.

Tests included. Help updated.

### CODE REVIEW

File: src/ui/actions/UnlabelAllTasksAction.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//
+// Created by Anton O. on 1/17/22.
+//
+
+#ifndef TASKMANAGER_SRC_UI_ACTIONS_UNLABELALLTASKSACTION_H
+#define TASKMANAGER_SRC_UI_ACTIONS_UNLABELALLTASKSACTION_H
+
+#include "Action.h"
+
+class UnlabelAllTasksAction : public Action {
```
Comment (KharchenkoMaks) [inline src/ui/actions/UnlabelAllTasksAction.h]: Confusing class name

**Review by KharchenkoMaks — CHANGES_REQUESTED** [review]

File: src/ui/actions/UnlabelAllTasksAction.h (hunk 1 above)
Comment (antonovchtd) [reply src/ui/actions/UnlabelAllTasksAction.h]: renamed UnlabelAllTasksAction to ClearAllLabelsOfTaskAction
UnlabelTaskAction to ClearLabelOfTaskAction
GetTasksToShowLabelsAction to GetTaskToShowLabelsAction

**Review by MichaelKupriianov — APPROVED** [review]

**Review by MichaelKupriianov — CHANGES_REQUESTED** [review]
.

File: src/ui/actions/ClearAllLabelsOfTaskAction.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+//
+
+#ifndef TASKMANAGER_SRC_UI_ACTIONS_UNLABELALLTASKSACTION_H
+#define TASKMANAGER_SRC_UI_ACTIONS_UNLABELALLTASKSACTION_H
+
+#include "Action.h"
+
+class ClearAllLabelsOfTaskAction : public Action {
+public:
+    ClearAllLabelsOfTaskAction(const Core::TaskID &);
```
Comment (AntonApraksin) [inline src/ui/actions/ClearAllLabelsOfTaskAction.h:12]: Mark explicit

File: src/ui/steps/LabelsStep.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
+//
+// Created by Anton Ovcharenko on 18.01.2022.
+//
+
+#include "LabelsStep.h"
+#include "ui/Context.h"
+
+std::unique_ptr<Action> LabelsStep::genAction(Context &context) {
+    auto labels = context.tasks()[0].data().labels();
```
Comment (AntonApraksin) [inline src/ui/steps/LabelsStep.cpp:9]: Unchecked access

File: src/ui/steps/UnlabelAllStep.cpp [hunk 4]
```diff
@@ -0,0 +1,15 @@
+//
+// Created by Anton O. on 1/17/22.
+//
+
+#include "UnlabelAllStep.h"
+#include "ui/Context.h"
+
+std::unique_ptr<Action> UnlabelAllStep::genAction(Context &context) {
+    return std::unique_ptr<Action>(new ClearAllLabelsOfTaskAction(*context.id()));
```
Comment (AntonApraksin) [inline src/ui/steps/UnlabelAllStep.cpp:9]: Unchecked access

**Review by AntonApraksin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**